src/components/Sections/SectionChart/SectionBarChart.js

```diff
@@ -193,7 +193,12 @@ const SectionBarChart = ({ data, style, dimensions, legend, chartProperties = {}
       }
       return dataItem;
     });
+
+    if (stacked) {
```

Comment:
why we cannot leave it in the original place ?

Reply:
the code inside the if(legend) modify the order

Reply:
the code inside the if shouldn't change the order

Reply:
this is the equivalent for the code in the web-client:
```
allSubGroups = Object.values(allSubGroups).sort((a, b) => sortByField(['sum', 'name'], [false, true])(a, b));
```